<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>...</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>item1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item3</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBox_2">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="toolButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBox_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>ITEMA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITEMB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITEMC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ITEMD</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>list</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>item1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>tree</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string>All Items</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>itemA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>itemB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>itemC</string>
            </property>
            <item>
             <property name="text">
              <string>subItem01</string>
             </property>
             <item>
              <property name="text">
               <string>001</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>subItem02</string>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>ItemD</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>table</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>ColA</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ColB</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ColC</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ColD</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>some</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>thing</string>
            </property>
           </item>
           <item row="0" column="2">
            <property name="text">
             <string>just </string>
            </property>
           </item>
           <item row="0" column="3">
            <property name="text">
             <string>show</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QSlider" name="verticalSlider">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0" colspan="2">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_kk">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
       <item row="6" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="layout_placehold"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuQss_file">
     <property name="title">
      <string>Qss file</string>
     </property>
     <addaction name="actionload"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuQss_file"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_qss">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit_qss"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="toolTip">
        <string>Apply stylesheet</string>
       </property>
       <property name="statusTip">
        <string>Apply stylesheet to application</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_buttons">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="button_common">
       <property name="text">
        <string>Common</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_mainwindow">
       <property name="text">
        <string>MainWindow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_dialog">
       <property name="text">
        <string>Dialog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_midarea">
       <property name="text">
        <string>MDIArea</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_mdisubwindow">
       <property name="text">
        <string>MDISubwindow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_dockwidget">
       <property name="text">
        <string>DockWidget</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_statusbar">
       <property name="text">
        <string>QStatusBar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_menubar">
       <property name="text">
        <string>QMenuBar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_menu">
       <property name="text">
        <string>QMenu</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>513</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="actionload">
   <property name="text">
    <string>load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
